frontend/src/js/controllers/featuredChallengeCtrl.js

```diff
@@ -30,19 +30,9 @@
         vm.loginContainer = angular.element('.exist-team-card');
```

Comment:
Hi, @aka-jain I don't understand this, this line is not changed in this PR. 
Is there a need to introduce this change here?